lib/Github/Api/Issue/Comments.php

```diff
@@ -23,8 +23,8 @@ class Comments extends AbstractApi
      */
     public function configure($bodyType = null)
     {
-        if (!in_array($bodyType, array('raw', 'text', 'html'))) {
-            $bodyType = 'full';
```

Comment:
Yes, I saw your comments. 

I think the best solution is to accept that we made a mistake before by keep the default body type to `full` for the comments API. In general we should align with the docs. 

Making a new major release just for this fix is not a good solution. What we should do is to create a new issue suggesting this change and add that issue in a 3.0 milestone. 

So in short: Just change this back to `full` and I'll be happy to merge.